llvm/tools/llvm-readobj/ELFDumper.cpp

```diff
@@ -4888,6 +4920,34 @@ void ELFDumper<ELFT>::printRelocationsHelper(const Elf_Shdr &Sec) {
 
 template <class ELFT> void ELFDumper<ELFT>::printDynamicRelocationsHelper() {
   const bool IsMips64EL = this->Obj.isMips64EL();
+  auto DumpCrelRegion = [&](DynRegionInfo &Region) {
```

Comment:
The lambda feels unnecessary, since it's only used once?

Reply:
It's used twice . There is `if (this->DynPLTRelRegion.Size) {` below